base_layer/core/src/mining/miner.rs

```diff
@@ -79,7 +79,7 @@ impl<B: BlockchainBackend> Miner<B> {
         Miner {
             kill_flag: stop_flag,
             consensus,
-            received_new_block_flag: Arc::new(AtomicBool::new(false)),
+            received_new_block_flag: Arc::new(AtomicBool::new(true)),
```

Comment:
👁 How did mining work before??

Reply:
Its not supposed to start automatically. This will break it. 
The miner waits until the base node is synced and ready for new blocks before mining. 
The miner stops and waits for a a new block event before it starts mining. Thats triggered by the events from the base node. This should be false

Reply:
Ok, I've added docs to explain this more unambiguously and reset the flag to false.